seqr/views/apis/data_manager_api_tests.py

```diff
@@ -743,11 +760,45 @@ def test_load_rna_seq_sample_data(self, mock_logger, mock_open, mock_os):
 
                 mock_open.assert_called_with(file_name, 'rt')
 
-                mock_logger.info.assert_has_calls([
-                    mock.call('Loading outlier data for NA19675_D2', self.data_manager_user),
-                    mock.call(f'create 2 {model_cls.__name__}', self.data_manager_user, db_update={
-                        'dbEntity':  model_cls.__name__, 'numEntities': 2, 'parentEntityIds': [RNA_SAMPLE_GUID], 'updateType': 'bulk_create',
-                    }),
-                ])
+                mock_logger.info.assert_called_with('Loading outlier data for NA19675_D2', self.data_manager_user)
+                mock_model_logger.info.assert_called_with(
+                    f'create 2 {model_cls.__name__}s', self.data_manager_user, db_update={
+                        'dbEntity': model_cls.__name__, 'entityIds': mock.ANY, 'parentEntityIds': [RNA_SAMPLE_GUID],
+                        'updateType': 'bulk_create',
+                    }
+                )
 
                 self.assertListEqual(params['get_models_json'](models), params['expected_models_json'])
+
+    @mock.patch('seqr.views.utils.dataset_utils.file_iter')
+    @mock.patch('seqr.views.apis.data_manager_api.logger')
+    @mock.patch('seqr.models.logger')
+    def test_load_phenotype_prioritization_data(self, mock_model_logger, mock_logger, mock_file_iter):
+        url = reverse(load_phenotype_prioritization_data)
+        self.check_data_manager_login(url)
+
+        mock_file_iter.return_value = iter(PHENOTYPE_PRIORITIZATION_MISS_HEADER)
+        response = self.client.post(url, content_type='application/json', data=json.dumps({'file': 'lirical_data.tsv.gz'}))
+        self.assertEqual(response.status_code, 500)
+        self.assertEqual(response.json()['error'], 'Invalid file: missing column(s) project, diseaseId')
+        mock_logger.info.assert_called_with('Loading phenotype-based prioritization data from lirical_data.tsv.gz', self.data_manager_user)
+        mock_file_iter.assert_called_with('lirical_data.tsv.gz')
+
+        mock_file_iter.return_value = iter(PHENOTYPE_PRIORITIZATION_HEADER + LIRICAL_NO_PROJECT_DATA)
+        response = self.client.post(url, content_type='application/json', data=json.dumps({'file': 'lirical_data.tsv.gz'}))
+        self.assertEqual(response.status_code, 500)
```

Comment:
never test a 500 error. 500 errors happen in an unhandled situation that gives me an urgent push notification, they should only happen when something is broken. If there is something that we would actually expect to happen as part of regular application usage, it needs to be a 400 error. If we really don't think this will happen, don't. test it